plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditorBase.java

```diff
@@ -149,6 +149,7 @@ public abstract class SQLEditorBase extends BaseTextEditor implements DBPContext
     private SQLCompletionContext completionContext;
     private SQLOccurrencesHighlighter occurrencesHighlighter;
     private SQLSymbolInserter sqlSymbolInserter;
+    protected DBPDataSourceContainer dataSourceContainer;
```

Comment:
We shouldn't move it here. Container is usually calculated dynamically in sub classes.